packages/runtime/src/state/actions.ts

```diff
@@ -99,10 +99,15 @@ type DocumentPayloadBaseDocument = {
   locale?: string | null // older versions of the runtime may not provide this field
 }
 
-type DocumentPayloadEmbeddedDocument = DocumentPayloadBaseDocument & {
+type DocumentPayloadEmbeddedDocument = {
+  key: string
+  locale?: string | null // older versions of the runtime may not provide this field
   id: string
   type: string
   name: string
+  rootElement: Element | null
+  initialRootElement: Element
+  hasFallback?: boolean
   __type: typeof EMBEDDED_DOCUMENT_TYPE
 }
```

Comment:
This is a breaking change in the data type being transacted between the host and builder. Is it permissible to make this change since embedded components are not being used?

Reply:
Not used by users? AFAIK, people are using it for the one-click catalyst demo.

Reply:
Ah, let me rephrase: "not being used in general acceptable". Since the OCC demo users are limited, is it acceptable to tell them to upgrade their runtime instead of having to support backwards compatibility?

Reply:
I'm not sure about that. Let's talk about this during stand-up.

Reply:
What did you conclude after chatting with stakeholders @arvinpoddar?

Reply:
@migueloller Chatted with Nathan, and the answer is **it should be possible** to force all OCC hosts to update. We'd have to do it with a script that crawls the existing projects and redeploy them. Given that there are nearly 200 OCC hosts that would be affected, I didn't feel great about this option - I would've been okay if it had been a few dozen. As much as I didn't want backwards compatibility to be a concern so early, I don't see another way.